studio/src/pages/RequestDetailsPage/RequestDetailsPageV2/useOrphanLogs.ts

```diff
@@ -20,7 +20,12 @@ export function useOrphanLogs(traceId: string, spans: Array<OtelSpan>) {
             }
             // TODO - Use a more deterministic ID - preferably string that includes the trace+span+event_index
             const logId = Math.floor(Math.random() * 1000000);
-            const orphanLog = convertEventToOrphanLog(traceId, logId, event);
+            const orphanLog = convertEventToOrphanLog(
+              traceId,
+              logId,
+              event,
+              span.span_id,
```

Comment:
This is a change for the future, where we want to be able to link back to the related spans from log messages.